radish/matcher.py

```diff
@@ -83,6 +85,10 @@ def match_step(sentence, steps):
         if isinstance(pattern, re._pattern_type):  # pylint: disable=protected-access
             match = pattern.search(sentence)
             argument_match = RegexStepArguments(match)
+            if match:
+                longest_group = get_longest_group(match.regs)
+            else:
+                longest_group = 0
```

Comment:
I could if you would rather, but I would end up initializing it twice for no reason, since if there is a match it would be to that value and if not it is zero